src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -243,26 +223,29 @@ def _motion_exclusion(roi: DenseROI, movie_shape: Tuple[int, int]) -> bool:
     Parameters
     ----------
     roi: DenseROI
-        the ROI to check
+        The ROI to check
     movie_shape: Tuple[int, int]
         The frame shape of the movie from which ROIs were extracted in order
         of: (height, width).
 
     Returns
     -------
     valid: bool
-        whether this ROI is valid on motion exclusion
+        True, if the ROI in question does not interesect with a motion
+        exclusion area. False if it does.
 
     """
-    furthest_right_pixel = roi['x'] + roi['width'] - 1
-    furthest_down_pixel = roi['y'] + roi['height'] - 1
-    right_limit = movie_shape[1] - roi['max_correction_right']
-    bottom_limit = movie_shape[0] - roi['max_correction_down']
-
-    valid = ((roi['x'] > roi['max_correction_left'] - 1) &
-             (roi['y'] > roi['max_correction_up'] - 1) &
-             (furthest_right_pixel < right_limit) &
-             (furthest_down_pixel < bottom_limit))
+    # A rightward shift increases the min 'valid' left border of the movie
```

Comment:
yikes! I had that completely backwards. These names are counter-intuitive to me, thanks for adding the comment strings.

Reply:
Yeah, it was pretty confusing. One way I wrapped my head around it was by reasoning:

If I have a move an ROI that is on the very left edge of a movie 10 frames to the right in order to align it with itself on a different frame, then everything to the left of it can't have aligned with anything.